test/built-ins/DataView/prototype/byteLength/detached-buffer.js

```diff
@@ -3,12 +3,11 @@
 
 /*---
 esid: sec-get-dataview.prototype.bytelength
-description: Throws a TypeError if the instance has a detached buffer
+description: Returns 0 if the buffer is detached
 info: |
-  24.2.4.2 get DataView.prototype.byteLength
+  get DataView.prototype.byteLength
```

Comment:
Ok, thanks for calling this out and providing clear guidance here.

Reply:
Ok, just double checking this: 

> We're actually not changing DataView in the current PR, because engines are aligned with the spec

That's not true for Chrome, which is why I assumed these would be updated as well, and also why I posted this: https://github.com/tc39/ecma262/pull/2164#issuecomment-700052500

Reply:
Oops, you're right, I should have said "because engines *aren't* aligned upon *non*-spec behavior". I think this is still what we want for now, but I'll let @syg confirm that we're on the same page.